examples/rps-demo/Makefile

```diff
@@ -25,7 +25,10 @@ build_all: contract
 
 CONTRACT_OUTPUTS=$(BUILD_DIR)/RockPaperScissors.abi $(BUILD_DIR)/RockPaperScissors.bin $(BUILD_DIR)/RockPaperScissorsFactory.abi $(BUILD_DIR)/RockPaperScissorsFactory.bin $(BUILD_DIR)/dapp-contract.js
 
-contract: $(CONTRACT_OUTPUTS)
+$(BUILD_DIR):
+	@mkdir -p $(BUILD_DIR)
+
+contract: $(BUILD_DIR) $(CONTRACT_OUTPUTS)
 $(CONTRACT_OUTPUTS): dapp-contract.sol
```

Comment:
it's `$(CONTRACT_OUTPUTS)` not `contract` that should depend on `$(BUILD_DIR)`. Alternatively, and maybe simpler and better, just put the `mkdir -p` as the first command there.

Reply:
Also, we use `$(HIDE)` instead of `@`, a pattern set by Paul in the legicash-facts `Makefile`

Reply:
Okay, thanks - I did the latter since the former resulted in errors when I tried it, and I've switched to `$(HIDE)` as well.